prow/config/agent.go

```diff
@@ -84,6 +85,10 @@ func (ca *Agent) Start(prowConfig, jobConfig string) error {
 			} else {
 				skips = 0
 				ca.Lock()
+				delta := ConfigDelta{*ca.c, *c}
+				for _, subscription := range ca.subscriptions {
+					subscription <- delta
```

Comment:
An unbuffered channel here for a subscriber would allow a subscriber to DOS the config-agent, right? We could just throw this into a `go func() { ... }` and make it best-effort?

Reply:
Good point. You could Subscribe to create a buffered channel instead of passing it. 
I think you right you would need to use a goroutune, but of that form go func(chan) {}(chan) :) (https://github.com/golang/go/wiki/CommonMistakes)